lib/mauth/client.rb

```diff
@@ -97,9 +97,7 @@ def initialize(config = {})
           Rails.logger
         else
           require 'logger'
-          is_win = RUBY_PLATFORM =~ /mswin|windows|mingw32|cygwin/i
-          null_device = is_win ? 'NUL' : '/dev/null'
-          ::Logger.new(File.open(null_device, File::WRONLY))
```

Comment:
What changed that this is no longer needed?

Reply:
rubocop doesn't like `NUL`  and `/dev/null`:
https://github.com/mdsol/mauth-client-ruby/actions/runs/12077251048/job/33679854702?pr=70
```
lib/mauth/client.rb:101:34: C: [Correctable] Style/FileNull: Use File::NULL instead of NUL.
          null_device = is_win ? 'NUL' : '/dev/null'
                                 ^^^^^
lib/mauth/client.rb:101:42: C: [Correctable] Style/FileNull: Use File::NULL instead of /dev/null.
          null_device = is_win ? 'NUL' : '/dev/null'
                                         ^^^^^^^^^^^
```